File: Back/src/ProEventos.Domain/Models/Palestrante.cs
namespace ProEventos.Domain.models
{
     public class Palestrante
     {
        public int Id { get; set; }
        public int Nome { get; set; }
        public string MiniCurriculo { get; set; }
        public string FotoUrl { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public IEnumerable<RedeSocial> RedeSociais {get;set;}

     }
}

File: src/app/models/Palestrante.ts

import { Evento } from "./Evento";
import { RedeSocial } from "./RedeSocial";

export interface Palestrante {
    
    id: number; 
    nome: string; 
    miniCurriculo: string; 
    fotoUrl: string; 
    telefone: string; 
    email: string; 
    redeSociais: RedeSocial[]; 
    palestranteEventos: Evento[];
}
